### PR TITLE
Improve wording of pack quantities

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Improves the wording of pack behavior management. Sometimes even we struggle to see what the options mean.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | 
| UI Tests          | https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/8186337386
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   |

### CODE REVIEW

**Comment by Hlavtox** [comment]
Ping @kpodemski @MatShir @Julievrz

**Review by boherm — DISMISSED** [review]

**Review by FabienPapet — DISMISSED** [review]

**Comment by Julievrz** [comment]
Hi @Hlavtox 
What do you think of taking inspiration from the "When out of stock" section just below on the page?

Here's an example: 

Screenshot 2024-02-23 at 10 33 28

Also, what do you think of adding the "Edit default behavior" link in this section too?

**Comment by Hlavtox** [comment]
@Julievrz Yes, we could add the link. But I wanted to specifically name it "stock behavior" or "how to calculate stock", because right now it just says how the quantity will be decreased, but it ALSO/MAINLY means HOW the stock is calculated in the first place.

This configuration value means both:
- How the quantity is calculated.
- What will be decreased after order.

Full explanation here: https://github.com/PrestaShop/PrestaShop/issues/35367#issuecomment-1956145179

File: src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/StockType.php [hunk 1]
```diff
@@ -57,17 +57,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('pack_stock_management', ChoiceType::class, [
                 'label' => $this->trans(
-                    'Default pack stock management',
+                    'Default pack stock behavior',
                     'Admin.Shopparameters.Feature'
                 ),
                 'help' => $this->trans(
-                    'When selling packs of products, how do you want your stock to be calculated?',
+                    'When selling packs of products, how do you want the stock to be calculated by default? You can decide, if you want to manage the stocks of packs manually, or if it should be calculated automatically depending on stock of the products inside. You can further configure it for each product.',
```
Comment (Julievrz) [inline src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/StockType.php]:
```suggestion
                    'For packs of products, stocks can be calculated manually or automatically based on individual product stock levels. Further customization is possible for each product.',
```

**Comment by Julievrz** [comment]
> @Julievrz Yes, we could add the link. But I wanted to specifically name it "stock behavior" or "how to calculate stock", because right now it just says how the quantity will be decreased, but it ALSO/MAINLY means HOW the stock is calculated in the first place.
> 
> This configuration value means both:
> 
> * How the quantity is calculated.
> * What will be decreased after order.
> 
> Full explanation here: [#35367 (comment)](https://github.com/PrestaShop/PrestaShop/issues/35367#issuecomment-1956145179)

Thanks, it's way clearer with the full explanation. :)

**Comment by Hlavtox** [comment]
@Julievrz Wording applied. I also added the link to edit default settings, that's a good idea. :-)

Snímek obrazovky 2024-03-05 105758

**Review by MatShir — DISMISSED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by Hlavtox** [comment]
Auto test - https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/8154574648

**Review by kpodemski — DISMISSED** [review]
It seems that e2e tests have to be adapted to the new wording.

**Comment by Hlavtox** [comment]
New run https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/8186337386

**Comment by Hlavtox** [comment]
@kpodemski All green, can we merge?

**Review by nesrineabdmouleh — APPROVED** [review]

**Comment by Hlavtox** [comment]
Ping @kpodemski @boherm for re-review 🙏

**Review by boherm — APPROVED** [review]

**Pull request merged**